File: new-crs-sarif/test-project/cpp/vulnerable.cpp
#include <cstdlib>
#include <iostream>

int main() { // Command injection vulnerability
  char cmd[100];
  std::cout << "Enter command: ";
  std::cin >> cmd;
  system(cmd);

  int *ptr = new int(10);
  delete ptr;
  // Use-after-free vulnerability
  std::cout << "Value after delete: " << *ptr << std::endl;

  return 0;
}

File: new-crs-sarif/test-project/python/vulnerable.py
from flask import Flask, request
import subprocess

app = Flask(__name__)

@app.route("/run")
def run():
    cmd = request.args.get("cmd")
    # This is a command injection vulnerability
    subprocess.run(cmd, shell=True)
    return "OK"

if __name__ == "__main__":
    app.run()
